src/main/java/frc/robot/Constants.java

```diff
@@ -199,7 +192,7 @@ public static class Arm {
         public static final Translation2d ELBOW_SEGMENT_CENTER_OF_MASS = new Translation2d(
                 Units.inchesToMeters(16), 0);
         public static final Translation2d WRIST_SEGMENT_CENTER_OF_MASS = new Translation2d(
-                Units.inchesToMeters(9.5), 0);
+                Units.inchesToMeters(10.5), 0);
```

Comment:
We estimated 10.5 for the CG, but it is `(5.5*9.5+1.2*22)/6.7 = 11.74`

Reply:
Fixed in code